Solutions/MimecastAudit/Analytic Rules/MimecastAudit.yaml

```diff
@@ -0,0 +1,51 @@
+id: 9c5dcd76-9f6d-42a3-b984-314b52678f20
```

Comment:
Can you please follow the example analytical rule here to create your analytics template:
https://github.com/Azure/Azure-Sentinel/blob/master/Detections/SigninLogs/FailedLogonToAzurePortal.yaml

Reply:
There is only one analytic rule at this point. I am not sure what you want me to do and what is wrong with the rule.

Reply:
> Can you please follow the example analytical rule here to create your analytics template: https://github.com/Azure/Azure-Sentinel/blob/master/Detections/SigninLogs/FailedLogonToAzurePortal.yaml

@aprakash13 , please review for it and approve the change, thanks.

Reply:
@aprakash13 @v-sabiraj Please, we would need an update on this not being resolved, since it's blocking the resolution of the pull request. Thank you.